label_studio_ml/utils.py

```diff
@@ -59,10 +60,40 @@ def get_choice(completion):
 
 
 def get_image_local_path(url, image_cache_dir=None, project_dir=None, image_dir=None):
-    return get_local_path(url, image_cache_dir, project_dir, get_env('HOSTNAME'), image_dir)
+    image_local_path = get_local_path(url, image_cache_dir, project_dir, get_env('HOSTNAME'), image_dir)
```

Comment:
I'm worried that there might be some confusion between the model hostname and the Label Studio hostname.